cmd/create-efs-volume/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	k8sflag "k8s.io/component-base/cli/flag"
+	"k8s.io/component-base/logs"
+
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+
+	"github.com/openshift/aws-efs-csi-driver-operator/pkg/efscreate"
+	"github.com/openshift/aws-efs-csi-driver-operator/pkg/version"
+)
+
+func main() {
+	rand.Seed(time.Now().UTC().UnixNano())
+
+	pflag.CommandLine.SetNormalizeFunc(k8sflag.WordSepNormalizeFunc)
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+
+	logs.InitLogs()
+	defer logs.FlushLogs()
+
+	command := NewOperatorCommand()
+	if err := command.Execute(); err != nil {
+		fmt.Fprintf(os.Stderr, "%v\n", err)
+		os.Exit(1)
+	}
+}
+
+func NewOperatorCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-efs-volume",
+		Short: "OpenShift AWS EFS Create EFS Volume",
+		Run: func(cmd *cobra.Command, args []string) {
+			cmd.Help()
+			os.Exit(1)
+		},
+	}
+
+	ctrlCmd := controllercmd.NewControllerCommandConfig(
```

Comment:
Low prio: this looks like an overkill to me - it will do leader election, set up metrics and whatnot.

Reply:
we could disable those easily. I am leaning towards disabling those rather than having another way of starting this thing up. what do you think?